### DISCUSSION TITLE
9.%E8%BE%93%E5%87%BA/hexo/bd4a2d1f29a661ef512c1f3149b4dbbf939e68ef

### DISCUSSION THREAD

**Discussion by giscus** [AINightCoder/quartz#5] [Announcements]
# 9.%E8%BE%93%E5%87%BA/hexo/bd4a2d1f29a661ef512c1f3149b4dbbf939e68ef

双链测试 test_note1 test_note2 test_note2 Markdown 示例 Markdown简易入门教程 使用方式：打开 Source Mode 就可以看到所有markdown格式，照着来用就行了 标题 # 标题一 ## 标题二 ### 标题三 #### 标题四 ##### 标题五 ###### 标题六 文本 粗体 引用 中划线 ...

https://bs2h.xyz/9.%E8%BE%93%E5%87%BA/hexo/bd4a2d1f29a661ef512c1f3149b4dbbf939e68ef

**Comment by AINightCoder** [comment]
test comment
